appveyor.yml

```diff
@@ -35,11 +46,12 @@ for:
       - job_name: Windows-x64-debug
 
   before_build:
+  - if exist %BUILD_DIR% rd /q /s %BUILD_DIR% 2> NUL
```

Comment:
I don't agree with this change, AppVeyor is very slow without the cache.

Could you explain your reasoning for wanting to build on 2017?

Reply:
>I don't agree with this change, AppVeyor is very slow without the cache.

Huh... I put it there for the same reason you did, the first build breaks when you change image.

>Could you explain your reasoning for wanting to build on 2017?

Even if VS 2017 supports all of C++17, supporting the compiler means supporting their non-standard quirks and bugs as well. VS 2013 was particularly bad for this and I hope they've improved since. But in general, we can't know for sure that a change will compile if it isn't in CI.